### PR TITLE
Issue 597 - Allow embedding flippingbook

### LINKED ISSUE #597: [Embed] Flipbook embeds do not work
Reference: https://www.moleculardevices.com/en/assets/brochures/dd/img/a-life-sciences-company
Franklin: https://main--moleculardevices--hlxsites.hlx.page/en/assets/brochures/dd/img/a-life-sciences-company

### PR DESCRIPTION
**Pull request by alexcarol**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #597 

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/brochures/dd/img/a-life-sciences-company
- After: https://issue-597--moleculardevices--hlxsites.hlx.page/en/assets/brochures/dd/img/a-life-sciences-company

- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/brochures/dd/img/3d-biology-drug-discovery-paradigm
- After: https://issue-597--moleculardevices--hlxsites.hlx.page/en/assets/brochures/dd/img/3d-biology-drug-discovery-paradigm

### CODE REVIEW

**Review by jose-correia — APPROVED** [review]

File: blocks/embed/embed.js [hunk 1]
```diff
@@ -52,7 +52,27 @@ export function embedCerosFrame(url) {
   return embedHTML;
 }
 
-const loadEmbed = (block, link, autoplay) => {
+function embedFlippingBook(url) {
+  return `
+<div class="flippingbook-mobile">
+    <a href="${url.href}" title="View Publication" target="_blank">
+        <img src="https://online.flippingbook.com/Thumbnail.aspx?url=${encodeURIComponent(url.href)}&size=400" alt="Flippingbook thumbnail" >
+        <div></div>
```
[the remark this answers is not in the document]
Comment (alexcarol) [reply blocks/embed/embed.js:60]: I'm using it in the css to embed the mobile image (as a background). An img tag might suffice, but I'm using the same approach as the moldev website. Do you want me to try the alternative?

[the remark this answers is not in the document]
Comment (mhaack) [reply blocks/embed/embed.js:60]: Would `.flippingbook-mobile > a:after` work? Or why background image? Why not render an img tag?

[the remark this answers is not in the document]
Comment (alexcarol) [reply blocks/embed/embed.js:60]: I just implemented the same approach as the original site, I'll do a quick test and lyk

[the remark this answers is not in the document]
Comment (alexcarol) [reply blocks/embed/embed.js]: Fixed, thanks for the feedback!

**Review by mhaack — APPROVED** [review]

**Pull request merged**